### PR TITLE
Improve documentation more

### PR DESCRIPTION
**Pull request by GiudGiud**
Add parameter groups
Refs #20535

Add some missing stubs

Fixes some typo

Mastodon patch is ready
https://github.com/idaholab/mastodon/pull/430

### CODE REVIEW

**Comment by GiudGiud** [comment]
All failures should be unrelated now.
Need a volunteer for reviewing among all the code owners

**Comment by GiudGiud** [comment]
@joshuahansel can you please review?

**Review by joshuahansel — CHANGES_REQUESTED** [review]

File: modules/ray_tracing/src/userobjects/ConeRayStudy.C
Comment (GiudGiud) [inline modules/ray_tracing/src/userobjects/ConeRayStudy.C:27]:
```suggestion
      "directions", "The directions(s) of the cone(s) (points down the center of each cone).");
```
```suggestion
      "directions", "The direction(s) of the cone(s) (points down the center of each cone).");
```

File: modules/contact/doc/content/source/constraints/CartesianMortarMechanicalContact.md
Comment (GiudGiud) [inline modules/contact/doc/content/source/constraints/CartesianMortarMechanicalContact.md:14]:
```suggestion
which are on both sides of the contact axis.
```

**Review by joshuahansel — DISMISSED** [review]

**Comment by GiudGiud** [comment]
ugh lemme remove this merge commit

**Review by dschwen — DISMISSED** [review]
I like the new parameter groups. Just for future reference, we now have the ability to add doc strings to individual moose enum items, just in case you're looking for more stuff to document...

**Comment by GiudGiud** [comment]
looks like I ll have to wait for a devel merge before being able to address the merge conflict. 
That s ok though

**Comment by GiudGiud** [comment]
> Just for future reference, we now have the ability to add doc strings to individual moose enum items, just in case you're looking for more stuff to document...

tempting...

**Review by dschwen — APPROVED** [review]
re-approve

**Comment by GiudGiud** [comment]
Thanks!

**Pull request merged**
